servlet-feature-pack/pom.xml

```diff
@@ -428,6 +439,36 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>2.10</version>
```

Comment:
this needs to be completely removed, as feature pack wont work like this.
it is maven dependency and such should be defined in module.xml provisioning plugin does the rest of the work